src/test/java/com/softserve/edu/greencity/ui/tests/createNewsTests/CNFunctionalityTests.java

```diff
@@ -0,0 +1,239 @@
+package com.softserve.edu.greencity.ui.tests.createNewsTests;
```

Comment:
Package name must be lowercase not camelCase. In your case createnewstests or newstests.create